src/Tools/Generator.php

```diff
@@ -76,6 +76,36 @@ public function processRoute(Route $route, array $rulesToApply = [])
         return $parsedRoute;
     }
 
+    protected function getBodyParameters(ReflectionMethod $method, array $tags)
+    {
+        /** @var ReflectionClass $cls */
+        $cls = collect($method->getParameters())
+            ->reduce(function ($carry, $param) use ($method) {
```

Comment:
I would suggest rewriting this function to `return null` directly, not a variable named `$carry`. Returning a variable gives anyone looking at the code the impression that the variable has some significance and is used somewhere, when it really isn't. You can change the function declaration to:
```php
function ($_, $param) {
```

Since naming a variable `_` is a common convention for "we don't need this".

Also, I'd suggest switching this to be imperative (using a for loop that `breaks` when it's found the FormRequest). In a method with a lot of parameters, this could save us some unnecessary loop iterations (and make my above points moot.😆)